src/nearbyinteraction.cs

```diff
@@ -0,0 +1,136 @@
+//
+// NearbyInteraction bindings
+//
+// Authors:
+//	Whitney Schmidt  <whschm@microsoft.com>
+//
+// Copyright 2020 Microsoft Inc.
+//
+
+using ObjCRuntime;
+using Foundation;
+using CoreFoundation;
+using System;
+using Vector3 = global::OpenTK.Vector3;
+
+namespace NearbyInteraction {
+
+    [NoWatch, NoTV, NoMac, iOS (14, 0)]
+    [Static]
+    interface NIErrorDomain
+    {
+        [Field ("NIErrorDomain")]
+        NSString Domain { get; }
+    }
```

Comment:
This needs to be in the `NIErrorCode` error enum like this 

https://github.com/xamarin/xamarin-macios/blob/65237b9af78aeb58095555f6a6e29b18ccb46629/src/fileproviderui.cs#L22-L29